app/models/email.rb

```diff
@@ -56,7 +56,7 @@ def self.email_available?(email, community_id)
   end
 
   def self.send_confirmation(email, community)
-    MailCarrier.deliver_later(PersonMailer.email_confirmation(email, community))
+    Delayed::Job.enqueue(EmailConfirmationJob.new(email.id, community.id), priority: 2)
```

Comment:
If I remember correctly there was some point why the mail carrier abstraction was put in place. The deliver_later is not implemented yet but it could be. Maybe @rap1ds remembers the details?